packages/api-clients/open-api/bffApi.yml

```diff
@@ -118,8 +118,8 @@ paths:
             default: []
           explode: false
         - in: query
-          name: producersIds
-          description: comma separated sequence of producers IDs
+          name: consumersIds
+          description: comma separated sequence of consumers IDs
```

Comment:
I think we have to do the opposite: this is the case where the front-end currently sets the `consumerIds` filter to a single value: the requester tenant id - so that's the filter we can remove. For this case we should leave only the filter on producers